SH4RE/SH4RE/Login/CreateAccount.swift

```diff
@@ -58,12 +68,20 @@ struct CreateAccount: View {
                     }
 
                     Spacer()
-                        .frame(height: screenSize.height * 0.25)
+                        .frame(height: screenSize.height * 0.1)
```

Comment:
why did u change this, does it look better?

Reply:
I added a name field so there was too much space